src/module_wrap.h

```diff
@@ -60,6 +60,12 @@ class ModuleWrap : public BaseObject {
   SET_MEMORY_INFO_NAME(ModuleWrap)
   SET_SELF_SIZE(ModuleWrap)
 
+  bool IsAllowedStrongObjectAtExit() const override {
+    // XXX: The garbage collection rules for ModuleWrap are *super* unclear.
+    // Do these objects ever get GC'd? Are we just okay with leaking them?
+    return true;
+  }
```

Comment:
I know the bot already pinged the team, but I’d be curious to know if @nodejs/modules knows more about this … especially for `ModuleWrap`s created by the `vm` module, creating non-GC-able objects can’t be ideal, right…?

Reply:
I thought they were all weak at one point?